data_models/environments/dev

```diff
@@ -0,0 +1,8 @@
+DJANGO_SECRET_KEY=!f3p74+hi5omz+)m((+8597&&01h-&bf(0#d1mwvak&clmv*%g
```

Comment:
This secret key will never be a secret again. What I have done for adage-server is to create a configuration file that contains all of the secrets and immediately added that file name to .gitignore. To have something to represent that file in source control, I created a template (just added `.template` to the end of the file name`) with all of the secrets blanked out by comments.

I guess this is "just" a developer key, but I hope your dev server isn't exposed to the internet.